packages/shared-components/src/Forms/SummaryPage.tsx

```diff
@@ -228,47 +227,25 @@ export function SummaryPage({ form, submission, translations, formUrl }: Props)
               {translate(TEXTS.grensesnitt.summaryPage.editAnswers)}
             </Link>
           </div>
-          {submissionMethod !== "digital" && (innsending === "KUN_PAPIR" || innsending === "PAPIR_OG_DIGITAL") && (
+          {(app === "bygger" || submissionMethod === "paper" || innsending === "KUN_PAPIR") && (
```

Comment:
Fiks logisk brist her. Dersom innsending "INGEN" så skal ikke knappen som tar bruker til "send-i-posten" dukke opp.

Reply:
Hva skjer på bygger hvis søknaden kun støtter digital?

Reply:
I byggeren er det ikke implementert mulighet for å velge papir eller digital innsending. Vi defaulter til papir dersom innsending="PAPIR_OG_DIGITAL", og dersom skjemaet har KUN_DIGITAL så vil en feilmelding vises ved trykk på "Neste"-knappen: "Digital innsending er ikke støttet ved forhåndsvisning i byggeren."